workspaces/bulk-import/.changeset/happy-zoos-tie.md

```diff
@@ -0,0 +1,6 @@
+---
+'@red-hat-developer-hub/backstage-plugin-bulk-import': minor
+'@red-hat-developer-hub/backstage-plugin-bulk-import-backend': minor
```

Comment:
Can you please add the common package here as well? I guess there was changes as, too:

```suggestion
'@red-hat-developer-hub/backstage-plugin-bulk-import-backend': minor
'@red-hat-developer-hub/backstage-plugin-bulk-import-common': minor
```

I approve this PR already so that you can merge it after you applied this small change. Thanks @AndrienkoAleksandr